pkg/server/nodemanager/node_controller.go

```diff
@@ -324,7 +324,6 @@ func (c *NodeController) finishNodeStart(node *api.Node) error {
 	// a describe instance here...
 	addresses, err := c.CloudClient.WaitForRunning(node)
 	if err != nil {
-		klog.V(2).Infof("Unhealthy wait for running, terminating node: %s", node.Name)
```

Comment:
Just double-checking that you wanted to remove this log line and we don't need it for diagnosing node start problems?

Reply:
Good catch! I went too far in removing debug lines added by me :+1: reverted